TestProjects/UniversalGraphicsTest_Foundation/ProjectSettings/TagManager.asset

```diff
@@ -17,8 +17,8 @@ TagManager:
   - VolumeLayer2
   - VolumeLayer3
   - VolumeLayer4
-  - 
-  - 
+  - Terrain
+  - Props
```

Comment:
Why is it needed?

Reply:
It's not, I've changed it to just use the existing UI tag so I don't need to any new ones.